great_expectations/cli/toolkit.py

```diff
@@ -427,10 +428,9 @@ def load_data_context_with_error_handling(
                 continuation_message=EXIT_UPGRADE_CONTINUATION_MESSAGE,
             )
             if not exception_occurred and increment_version:
-                context = DataContext(directory)
+                context = DataContext(context_root_dir=directory)
         return context
     except ge_exceptions.UnsupportedConfigVersionError as err:
-        directory = directory or DataContext.find_context_root_dir()
```

Comment:
Don't we still want this? In case execution didn't make it to 421?

Reply:
This is odd...  This was not supposed to have disappeared.  Checking.